packages/relay/tests/lib/eth.spec.ts

```diff
@@ -110,7 +110,7 @@ describe('Eth calls using MirrorNode', async function () {
     process.env.ETH_FEE_HISTORY_FIXED = 'false';
 
     // @ts-ignore
-    ethImpl = new EthImpl(sdkClientStub, mirrorNodeInstance, logger, '0x12a', cache);
+    ethImpl = new EthImpl(sdkClientStub, mirrorNodeInstance, logger, '0x12a', registry, cache);
```

Comment:
Q: what happened here? Was registry just missing or is this related to the change?

Reply:
This is related to  the change, I had  to inject Registry for metrics in order to  be  able to instantiate the SDKClient on-demand.